tasks/setup-dnf.yml

```diff
@@ -6,12 +6,33 @@
                           else cockpit_packages }}"
   when: cockpit_packages is defined
 
-- name: Ensure Cockpit Web Console packages are installed.
+- name: Get cockpit-bridge available version in "full" package install mode
+  when: cockpit_packages == "full"
+  changed_when: false
+  command: dnf repoquery --latest-limit=1 --queryformat '%{version}' cockpit-bridge
+  register: __cockpit_bridge_version
+
+- name: Initialize dynamic excludes
```

Comment:
We don't need two separate tasks for this - we can do it all in 1 task:

```yaml
- name: Ignore cockpit-pcp in "full" package install mode with Cockpit ≥ 326
  set_fact:
    cockpit_packages_dyn_exclude: "{{ ['cockpit-pcp']
      if cockpit_packages == 'full'
      and __cockpit_bridge_version.stdout is version('326', '>=')
      else [] }}"
```

Reply:
Thanks! Applied. I'm still learning the balance between declarative+bigger and imperative.